api/security/policy.go

```diff
@@ -0,0 +1,85 @@
+package security
+
+import (
+	"net/http"
+
+	"github.com/gobwas/glob"
+)
+
+// HasOne returns a gin handler that checks the request against the given role
+func (policy RolePolicy) HasOne(roles ...Role) bool {
+	for _, role := range roles {
+		if policy[role] {
+			return true
+		}
+	}
+	return false
+}
+
+// HasAll returns a gin handler that checks the request against the given role
+func (policy RolePolicy) HasAll(roles ...Role) bool {
+	for _, role := range roles {
+		if !policy[role] {
+			return false
+		}
+	}
+	return true
+}
+
+// ToSlice returns a slice of roles
+func (policy RolePolicy) ToSlice() (roles []Role) {
```

Comment:
Do you need a ToSlice or a String() method. I'm saying this because you're using this for logging purposes only.

Reply:
A structured slice is reusable AND loggable